### PR TITLE
Feature/dtgb 282

### PR DESCRIPTION
**Pull request by delrueba**
## Screenshots (if appropriate):
image
image
image
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have updated the documentation accordingly.
- [x] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/00-mixins/breakpoints/_breakpoints.scss [hunk 1]
```diff
@@ -24,6 +24,22 @@
   }
 }
 
+///
+/// Phablet breakpoint (see $bp-phablet defined in _vars.scss).
+/// This asumes a min-width as defined by the variable.
```
Comment (meirege) [inline styleguide/components/00-mixins/breakpoints/_breakpoints.scss]: assumes ;)

File: styleguide/components/41-organisms/image-gallery/_image-gallery.scss [hunk 2]
```diff
@@ -0,0 +1,133 @@
+.image-gallery {
+  @include clearfix();
```
Comment (meirege) [inline styleguide/components/41-organisms/image-gallery/_image-gallery.scss:2]: Is a clearfix really necessary?

File: styleguide/components/41-organisms/image-gallery/_image-gallery.scss [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+      left: 0;
+      width: 100%;
+      height: 100%;
+      content: '';
+      opacity: .7;
+    }
+  }
+
+  .show-more__content {
+    @include theme('color', 'color-secondary', 'image-gallery-show-more-color');
```
Comment (meirege) [inline styleguide/components/41-organisms/image-gallery/_image-gallery.scss:55]: New line plz.

File: styleguide/components/41-organisms/image-gallery/_image-gallery.scss [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+    i {
+      font-size: 1.5rem;
+      line-height: 1;
+      vertical-align: middle;
+    }
+  }
+
+  @include phablet {
+    // |____||____|
```
Comment (meirege) [inline styleguide/components/41-organisms/image-gallery/_image-gallery.scss]: Maybe add some text to this? ;)

**Review by meirege — CHANGES_REQUESTED** [review]
@delrueba Maybe add a preview template with some gallery examples in it?

File: styleguide/components/41-organisms/image-gallery/_image-gallery.scss (hunk 2 above)
Comment (delrueba) [reply styleguide/components/41-organisms/image-gallery/_image-gallery.scss:2]: Unfortunately yes, it all floats.

File: styleguide/components/00-mixins/breakpoints/_breakpoints.scss (hunk 1 above)
Comment (delrueba) [reply styleguide/components/00-mixins/breakpoints/_breakpoints.scss]: :open_mouth:

File: styleguide/components/41-organisms/image-gallery/_image-gallery.scss (hunk 3 above)
Comment (delrueba) [reply styleguide/components/41-organisms/image-gallery/_image-gallery.scss:55]: done

**Review by meirege — APPROVED** [review]

**Pull request merged**